platforms/core-configuration/declarative-dsl-provider/src/test/kotlin/org/gradle/internal/declarativedsl/settings/UnsupportedSyntaxFeatureCheckTest.kt

```diff
@@ -21,6 +21,7 @@ import org.gradle.internal.declarativedsl.analysis.AnalyzedStatementUtils
 import org.gradle.internal.declarativedsl.analysis.DefaultOperationGenerationId
 import org.gradle.internal.declarativedsl.analysis.analyzeEverything
 import org.gradle.internal.declarativedsl.analysis.tracingCodeResolver
+import org.gradle.internal.declarativedsl.common.UnsupportedSyntaxFeatureCheck
```

Comment:
Should the test be moved to the corresponding package, too?

Reply:
Fixed.